docs/problem_format/custom_checkers.md

```diff
@@ -116,24 +103,22 @@ The only built-in checker that has this flag set is the `linecount` checker.
 `check` can return either a `CheckerResult` object (`from dmoj.result import CheckerResult`), or a boolean (`case_passed_bool`).
  A `CheckerResult` can be instantiated through `CheckerResult(case_passed_bool, points_awarded, feedback='')`.
 
-# Checkers or Interactors for Computationally-Heavy Problems
-Sometimes, problems that require checkers or interactive grading may also be computationally expensive. In such cases, it is often beneficial to move answer checking from the slow Python problem module and into a native language. For maximum interoperability, the judge system requires a Python script to handle raw interaction prompts, but the script itself may use the judge's compilation and execution API to spawn native processes.
+## Native Checkers
+Sometimes, problems that require checkers may also be computationally expensive.
+In such cases, it is often beneficial to move answer checking from the slow Python problem module and into a native language.
 
-To illustrate, a problem that requires a computationally expensive validator can easily be implemented as shown below.
+This is handled by `bridged` checker.
 
-```python
-from dmoj.executors import executors
-from dmoj.judgeenv get_problem_root
-import os
-
-# Locate the validator source file. get_problem_root returns the root directory of the problem passed by name.
-validator_path = os.path.join(get_problem_root('problem_id'), 'validator.cpp')
-# Read all source into memory
-with open(validator_path, 'r') as validator_source:
-    # Execute the compiler - executors[] is a mapping of all executors by language id
-    executor = executors['CPP11'].Executor('validator', validator_source.read())
-# Launch the validator in a sandbox - see below
-process = executor.launch(time=60, memory=262144)
-```
+The `bridged` checker takes the following arguments:
+
+- `files`: either a filename, or a list of filenames, corresponding to the checker
```

Comment:
Be consistent in adding punctuation, if you add it on some bullet points, add it on all of them.